wom/src/main/scala/wdl4s/wdl/examples/ex7.scala

```diff
@@ -36,7 +36,7 @@ object ex7 {
 
     ns.taskCalls.find( _.unqualifiedName == "a") foreach { call =>
       val wdlFunctions: CustomFunctions = new CustomFunctions
-      val evaluatedInputs = call.evaluateTaskInputs(inputs, wdlFunctions).get
+      val evaluatedInputs = call.evaluateCallDeclarations(inputs, wdlFunctions).get
```

Comment:
Each file in this folder is a compiled check of a README example. Thus any updates here should have an accompanying change there.

In this case, the corresponding block is [here](https://github.com/broadinstitute/wdl4s/blob/814d20317b835e3bb785e57d9c17cac2676a1dbb/README.md#user-content-instantiating-commands).